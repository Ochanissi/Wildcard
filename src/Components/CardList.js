import React from 'react';
import Card from './Card';

const CardList = ({ characters }) => {
    return (
        <div>
            {
                characters.map((user, i) => {
                    return (
                        <Card 
                            key={i} 
                            id={characters[i].id} 
                            name={characters[i].name} 
                            email={characters[i].email}
                            />
                    );
                })
            }
        </div>
    );
}

export default CardList;