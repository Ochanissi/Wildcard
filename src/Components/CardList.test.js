import { shallow, mount, render } from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('expect to render CardList component', () => {
    const mockCharacters = [
        {
            id: 1,
            name: 'John Snow',
            username: 'JohnJohn',
            email: 'john@gmail.com'
        }
    ]
    expect(shallow(<CardList characters={mockCharacters} />)).toMatchSnapshot();
})