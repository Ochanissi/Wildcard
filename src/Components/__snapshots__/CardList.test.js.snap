// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `ShallowWrapper {}`;
